universal-application-tool-0.0.1/app/models/Program.java

```diff
@@ -1,58 +1,64 @@
 package models;
 
-import io.ebean.annotation.DbJsonB;
-import io.ebean.text.json.EJson;
-import java.io.IOException;
-import java.util.Map;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.google.common.collect.ImmutableList;
+import io.ebean.annotation.DbJson;
 import javax.persistence.Entity;
+import javax.persistence.PostLoad;
+import javax.persistence.PrePersist;
 import javax.persistence.Table;
 import play.data.validation.Constraints;
+import services.program.BlockDefinition;
+import services.program.ProgramDefinition;
 
 @Entity
 @Table(name = "programs")
 /** The ebeans mapped class for the program object. */
-public class Program {
-  private static final long serialVersionUID = 1L;
+public class Program extends BaseModel {
```

Comment:
nit: please move comment to above annotations here and in the two methods below

Reply:
Done